File: api/v2010/message/create-using-keys-example/create-using-keys-example.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
api_key = 'SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
api_secret = 'your_api_secret'
@client = Twilio::REST::Client.new(api_key, api_secret, account_sid)

message = @client.messages.create(
                             body: 'Never gonna give you up.',
                             from: '+15017122661',
                             to: '+15558675310'
                           )

puts message.sid

File: api/v2010/message/create-using-keys-example/create-using-keys-example.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid at twilio.com/console
        // Provision API Keys at twilio.com/console/runtime/api-keys
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string apiKey = "SKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string apiSecret = "your_api_secret";

        TwilioClient.Init(apiKey, apiSecret, accountSid);

        var message = MessageResource.Create(
            body: "Never gonna give you up.",
            from: new Twilio.Types.PhoneNumber("+15017122661"),
            to: new Twilio.Types.PhoneNumber("+15558675310")
        );

        Console.WriteLine(message.Sid);
    }
}
